great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -544,7 +544,7 @@ def _get_sqlalchemy_key_pair_auth_url(
 
     def get_domain_records(  # noqa: C901 - 24
         self,
-        domain_kwargs: Dict,
+        domain_kwargs: dict,
```

Comment:
In the case we remove all `Dict`, please remove the import as well

Reply:
@cdkini Thanks!  I think that the linter should tell me as much.  Checking now.

Reply:
@cdkini In this case, it seems that this import is used elsewhere.